crates/astria-composer/src/composer.rs

```diff
@@ -66,11 +90,13 @@ impl Composer {
         let Self {
             api_server,
             searcher,
+            executor_handle,
         } = self;
 
         let api_task =
             tokio::spawn(async move { api_server.await.wrap_err("api server ended unexpectedly") });
         let searcher_task = tokio::spawn(searcher.run());
+        let _ = executor_handle.send_bundles;
```

Comment:
if you added this to avoid the errors from an unused variable, you can name it `_executor_handle` to avoid getting that error

Reply:
yes sounds good. will do that